docs/general/manage-tokens/holding-rose-tokens/custody-providers.md

```diff
@@ -33,3 +34,17 @@ Finoa is a regulated custodian for digital assets, servicing professional invest
 * **Delegation options:** Finoa offers delegation to a number of select validators including Bison Trails, Blockdaemon, Chorus One, Figment Networks, and more.
 * **Min holding:** No threshold for assets under custody. Please check out details at [finoa.io](https://www.finoa.io).
 * **Sign up:** Email [oasis@finoa.io](mailto:oasis@finoa.io) to set up an account.
+
+### [Oasis Safe](https://safe.oasis.io)
```

Comment:
Actually listing it in this section makes it sound as if this is a custody provider? It is not, this is a non-custodial wallet where the owner holds the keys. Oasis Safe only runs some supporting infrastructure but it does not have access to any funds (e.g. in case the user loses all keys or bricks the smart contract configuration, there is no way to recover).

Reply:
Was pondering whether making a new chapter for this, but now decided to reorganize it as "Custody Providers & Protocols".

Reply:
Ah, I see this is Safe's terminology. I was initially thrown off but 👌